### PR TITLE
(DOCSP-7031): What is Realm? page

### PR DESCRIPTION
**Pull request by cbush**
- First pass at the What is Realm? page
- Merged intro into one page, no point being wordy
- "Realm Cloud" nomenclature not official, still TBD
- End of page links to be updated later
- Probably needs images or visual help

## Staged
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker/realm-intro/get-started/what-is-realm

## JIRA
https://jira.mongodb.org/browse/DOCSP-7031

### CODE REVIEW

File: source/get-started/what-is-realm.txt [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Designed for mobile development:
+  Mobile app developers must deal with unreliable networks,
+  limited bandwith, and finite power. Users expect a
+  responsive app experience in which they can read and write
+  data regardless of connectivity. Realm is specifically
+  designed to meet these challenges.
+
+Sync with security: 
+  Users expect security while selectively sharing synchronized
+  data with teammates, friends, or other user groups. Realm is
```
Comment (rkumar-mongo) [inline source/get-started/what-is-realm.txt]: Maybe add "Developers want to provide these features without dealing with the complexity of integration and upkeep

**Comment by rkumar-mongo** [comment]
Looking at this, I wonder if something like this might work:

- "Why Realm" as an H1
- "Realm Product Suite" as an H2

That way we start with a very brief hook/reminder of what Realm does for you, then let users see the list of available products and link to where they need to go to learn more. Cards might be nice here, I feel like Nick had a really nice POC a while back on that concept.

File: source/get-started/what-is-realm.txt [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+Realm Database:
+  - Provides a simple and fast alternative to SQLite and Core Data, available for Android, iOS, Node.js, React Native, Xamarin, and UWP development.
+  - Eliminates the need for ORMs and other boilerplate data transformation code by working directly with your objects across all supported languages.
+  - Simplifies the development of highly concurrent, reactive apps with its powerful threading model and notifications system.
+
+Realm Sync:
+  - Seamlessly synchronizes your data across devices.
+  - Allows your users to read and write data regardless of network connectivity.
+  - Guarantees strong eventual consistency in data with its built-in conflict resolution.
```
[the remark this answers is not in the document]
Comment (cbush) [reply source/get-started/what-is-realm.txt]: I don't know about "user data". It is all data.

File: source/get-started/what-is-realm.txt [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  Users expect security while selectively sharing synchronized
+  data with teammates, friends, or other user groups. Realm is
+  the only major solution today that scales well with both
+  Sync and access rules.
+
+Infrastructure:
+  Database and server administration and other infrastructure
+  pieces are critical and complex, but too many developers
+  spend too much time and effort there when these pieces are
+  not special to the app they are building. By handling the
```
[the remark this answers is not in the document]
Comment (cbush) [reply source/get-started/what-is-realm.txt]: Good point. I am trying it with "your app" still. Thoughts?

[the remark this answers is not in the document]
Comment (cbush) [reply source/get-started/what-is-realm.txt]: As if the reader has a team of developers working on "their" app.

"The app" seemed too "what app?"

**Comment by cbush** [comment]
Thanks @rkumar-mongo! I will be getting more boilerplate from product and looking into cards.

**Comment by cbush** [comment]
Thank you. My thoughts and intentions exactly.

**Pull request merged**